core/ntp.go

```diff
@@ -0,0 +1,68 @@
+package core
+
+import (
+	"context"
+	"sync/atomic"
+	"time"
+
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/beevik/ntp"
+)
+
+var (
+	ntpOffsetNano int64
```

Comment:
is there benefit in this approach, compared to a struct like `NTPSyncedClock`, where users only need to care about `NTPSyncedClock.Now()`?